src/style.css

```diff
@@ -0,0 +1,65 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  /* background-size: cover; */
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 :nth-child(n) {
+  background-image: url(images/star.svg);
+}
+
+.stars--1 :first-child {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--1 :not(:first-child) {
+  background-image: url(images/star.svg);
+}
+
+.stars--2 :nth-child(1),
+.stars--2 :nth-child(2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 :nth-child(3),
+.stars--2 :nth-child(4),
+.stars--2 :nth-child(5) {
+  background-image: url(images/star.svg);
+}
+
+.stars--3 :nth-child(1),
+.stars--3 :nth-child(2),
+.stars--3 :nth-child(3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 :nth-child(4),
+.stars--3 :nth-child(5) {
+  background-image: url(images/star.svg);
+}
+
+.stars--4 :last-child {
+  background-image: url(images/star.svg);
+}
+
+.stars--4 :not(:last-child) {
```

Comment:
rewrite all these code, it shoulkd look like that 
.stars--5 .stars__star,
.stars--1 .stars__star:first-child,
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:not(:last-child) {
  background-image: url(./images/star-active.svg);
}